### DISCUSSION TITLE
How can I change the icons?

### DISCUSSION THREAD

**Discussion by hm21** [hm21/pro_image_editor#43] [Q&A] [answered]
Asked [here](https://github.com/hm21/pro_image_editor/discussions/33#discussioncomment-9380847)

**Comment by hm21** [comment, marked as answer]
To change the icons, you have to set it in the configs. Below is a short example, and [here](https://github.com/hm21/pro_image_editor/blob/0a23cb80c03749a40293cf9b4284071b9149ae49/example/lib/pages/highly_configurable_example.dart) you can see the full example.

```dart
ProImageEditor.network(
      'https://picsum.photos/id/230/2000',
      onImageEditingComplete: onImageEditingComplete,
      onCloseEditor: onCloseEditor,
      configs: ProImageEditorConfigs(
              icons: const ImageEditorIcons(
                  paintingEditor: IconsPaintingEditor(
                    bottomNavBar: Icons.palette,
                    lineWeight: Icons.line_weight_rounded,
                    freeStyle: Icons.roundabout_left,
                    arrow: Icons.arrow_downward_rounded,
                    line: Icons.horizontal_rule,
                    fill: Icons.format_color_fill,
                    noFill: Icons.format_color_reset,
                    rectangle: Icons.crop_free,
                    circle: Icons.lens_outlined,
                    dashLine: Icons.power_input,
                  ),
     ),
)
```

**Reply by UmangPandav** [reply]
how to change icon color?

**Reply by hm21** [reply]
To change the icon color, you have two options.

Option 1:
You set a custom icon color inside the theme, which will change every icon color. 
As example:
```dart
configs: ProImageEditorConfigs(
      theme:ThemeData(iconTheme: const IconThemeData(color: Colors.yellow))
)
```

Option 2: 
You create your own AppBar/BottomBar which allows you to change everything in detail. You can check out the example [here](https://github.com/hm21/pro_image_editor/blob/0a23cb80c03749a40293cf9b4284071b9149ae49/example/lib/pages/custom_appbar_bottombar_example.dart) and read about the discussion [here](https://github.com/hm21/pro_image_editor/issues/38).

**Reply by UmangPandav** [reply]
Nice
